src/main.js

```diff
@@ -10,14 +11,16 @@ function toggleMenu() {
   displayOff();
   menuClose.classList.remove('hidden');
   menuOpen.classList.remove('hidden');
+  shadow.classList.add('shadow');
 }
 
+
 function displayBack() {
```

Comment:
We've really done a tremendous job with the display of the nav and its elements!  When we refactor this JS file, we should reduce some of our WET code so that this function, in addition to displayBack(), has a parameter that takes an argument of whichever node's classList we are looking to manipulate.